src/main/java/redis/clients/jedis/commands/AdvancedJedisCommands.java

```diff
@@ -64,5 +65,11 @@
 
   List<String> aclCat(String category);
 
+  List<Map<String,String>> aclLog();
+
+  List<Map<String,String>> aclLog(int limit);
```

Comment:
Same as line 70.

Reply:
I have created a custom object for the log entry see AccessControlLogEntry class